src/compiler/transformers/ts.ts

```diff
@@ -223,7 +225,7 @@ namespace ts {
          * @param node The node to visit.
          */
         function visitTypeScript(node: Node): VisitResult<Node> {
-            if (hasModifier(node, ModifierFlags.Ambient)) {
+            if (hasModifier(node, ModifierFlags.Ambient) && isStatement(node)) {
                 // TypeScript ambient declarations are elided, but some comments may be preserved.

```

Comment:
Looks like this expression can be simplified to `return createNotEmittedStatement(node)`